### DISCUSSION TITLE
Avalon template v6.0.0 components missing

### DISCUSSION THREAD

**Discussion by joe380** [primefaces/community#2586] [PrimeVue Templates]
Hello, 
I've noticed that the AVALON version 5.3.0 had a calendar (full-calendar) in the templates and the version 6.0.0 is missing this whole calendar page.
Was there a particular reason not to add it in the new version, any collisions, problems, because I will probably need that component later in the project.
It's visible in the preview: https://avalon.primevue.org/apps/calendar (old version)
I've built the 6.0.0 and it was missing between blog and chat menu.
<img width="409" alt="avalon1" src="https://github.com/user-attachments/assets/e40d50c0-70ec-41f7-8402-3d2b7b05124a">
<img width="636" alt="avalon2" src="https://github.com/user-attachments/assets/c1f4184f-be2b-4013-951d-2e9c1703d7f2">

Thx.

**Comment by tugcekucukoglu** [comment]
Calendar demo page is removed from Avalon.

Thank you.

**Reply by joe380** [reply]
Hello,
that it was removed, I can see, the more interesting question is why? The
calendar component was actually the reason why I both the template int the first
place, because it had the calendar, otherwise also sakai would suffice.
What is now the difference between sakai and any premium template?

**Reply by joe380** [reply]
Hello,
so any reason for removing calendar?
And If I need the calendar, can it be easily copied from previous avalon to the new avalon template?
Thx
